Client/Frontend/Browser/TabManager.swift

```diff
@@ -354,13 +354,14 @@ class TabManager : NSObject {
         assert(NSThread.isMainThread())
         // If the removed tab was selected, find the new tab to select.
         if tab === selectedTab {
-            if let index = getIndex(tab) {
-                if index + 1 < count {
-                    selectTab(tabs[index + 1])
+            let viableTabs: [Tab] = tab.isPrivate ? privateTabs : normalTabs
```

Comment:
The problem with having it as a computer property is that in only one of the three places that pattern appears, we can be certain it's the selected tab whose privacy we are checking — in others it's regarding a certain tab that's being interacted with, but might not actually be selected yet. We could make it a method, but it seems small enough that that might not worth it?